x-pack/plugins/uptime/server/lib/telemetry/queue.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const TELEMETRY_MAX_QUEUE_SIZE = 100;
```

Comment:
Do we know the size of the documents? In the Security Solutions implementation, this is done to reduce the chances of sending payloads larger than 10MB (which is the limit of our receiving API).

Reply:
There is definitely a lot of reuse here from Fleet's implementation actually. We are sending super small payloads, documents with only about 10 fields each.